marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/entities/search/buildPropertyMetadata.sjs

```diff
@@ -37,7 +39,8 @@ function verifySimpleProperty() {
     test.assertEqual("customerId", metadata[0].propertyPath),
     test.assertEqual("customerId", metadata[0].propertyLabel),
     test.assertEqual("integer", metadata[0].datatype),
-    test.assertEqual(false, metadata[0].multiple)
+    test.assertEqual(false, metadata[0].multiple),
```

Comment:
Isn't this missing an additional assertion for `test.assertEqual(true, metadata[0].sortable),`?